crates/uv-client/src/base_client.rs

```diff
@@ -112,7 +112,7 @@ impl<'a> BaseClientBuilder<'a> {
 
         // Timeout options, matching https://doc.rust-lang.org/nightly/cargo/reference/config.html#httptimeout
         // `UV_REQUEST_TIMEOUT` is provided for backwards compatibility with v0.1.6
-        let default_timeout = 5 * 60;
+        let default_timeout = 30;
```

Comment:
Is this documented anywhere?

Reply:
Looks like the 5 min were previously undocumented.

It's also a bit misleading since it's not an http timeout properly, it's just a read timeout.

Reply:
👍 I don't think we'll ever want a real timeout per http request, I think a read timeout makes sense. If we expose a comprehensive timeout, I think it'd expect it to be like for an entire command invocation or installation of a package or something.

Reply:
For what it's worth,

Poetry and Pip use 15 seconds as a default
* https://python-poetry.org/docs/faq/#my-requests-are-timing-out
* https://github.com/pypa/pip/blob/main/src/pip/_internal/cli/cmdoptions.py#L294